Testing/functional/tests/cypress/integration/ui/pages/vaccineCard.js

```diff
@@ -0,0 +1,545 @@
+    cy.get("[data-testid=phnInput]")
+        .should("be.visible", "be.enabled")
+        .type(phn);
+}
+
+function select(selector, value) {
+    cy.get(selector).should("be.visible", "be.enabled").select(value);
+}
+
+function selectExist(selector, value) {
+    cy.get(selector)
+        .children("[value=" + value + "]")
+        .should("exist");
+}
+
+function selectNotExist(selector, value) {
+    cy.get(selector)
+        .children("[value=" + value + "]")
+        .should("not.exist");
+}
+
+function clickVaccineCardEnterButton() {
+    cy.get("[data-testid=btnEnter]").should("be.enabled", "be.visible").click();
+}
+
+describe("Vaccine Card Page", () => {
+    it("Landing Page - Vaccine Card Button does not exist - Vaccine Status module disabled", () => {
+        cy.enableModules({});
+        cy.visit(homeUrl);
+        cy.get("[data-testid=btnVaccineCard]").should("not.exist");
+    });
+
+    it("Landing Page - Log In - Vaccine Status module disabled and vaccine card URL entered directly", () => {
+        cy.enableModules({});
+        cy.visit(vaccineCardUrl);
+
+        cy.get("[data-testid=BCSCBtn]").should("be.visible", "be.enabled");
+        cy.get("[data-testid=IDIRBtn]").should("be.visible", "be.enabled");
+        cy.get("[data-testid=KeyCloakBtn]").should("be.visible", "be.enabled");
+    });
+
+    it("Landing Page - Vaccine Card Button Exists - Vaccine Status module enabled", () => {
+        cy.enableModules(vaccinationStatusModule);
+        cy.visit(homeUrl);
+        cy.get("[data-testid=btnVaccineCard]").should("exist");
+    });
+
+    it("Landing Page - Vaccination Card - unauthenticated user", () => {
+        cy.enableModules(vaccinationStatusModule);
+        cy.visit(vaccineCardUrl);
+
+        cy.get("[data-testid=vaccineCardFormTitle]").should("be.visible");
+        cy.get("[data-testid=btnCancel]").should("be.visible");
+        cy.get("[data-testid=btnEnter]").should("be.enabled", "be.visible");
+        cy.get("[data-testid=btnPrivacyStatement]").should("be.visible");
+        cy.get("[data-testid=btnLogin]").should("be.enabled", "be.visible");
+    });
+
+    it("Vaccination Card - Cancel - unauthenticated user", () => {
+        cy.enableModules(vaccinationStatusModule);
+        cy.visit(vaccineCardUrl);
+
+        cy.get("[data-testid=btnCancel]").should("be.visible").click();
+
+        cy.get("[data-testid=btnVaccineCard]").should("exist");
+    });
+
+    it("Vaccination Card - Login BC Services Card App - unauthenticated user", () => {
+        cy.enableModules(vaccinationStatusModule);
+        cy.visit(vaccineCardUrl);
+
+        cy.get("[data-testid=btnLogin]")
+            .should("be.visible", "be.enabled")
+            .click();
+
+        cy.get("[data-testid=BCSCBtn]").should("be.visible", "be.enabled");
+        cy.get("[data-testid=IDIRBtn]").should("be.visible", "be.enabled");
+        cy.get("[data-testid=KeyCloakBtn]").should("be.visible", "be.enabled");
```

Comment:
Same issue mentioned as mentioned above - suggest swapping these out for something more generic to make sure we have left the page.

Reply:
Changed.